src/sp/index.html

```diff
@@ -0,0 +1,181 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="../../css/scraper.css" />
+        <title></title>
+    </head>
+    <body>
+        <div class="header">
+            <a href="/">
+                <div class="logo">
+                    <h3><i>Scraper-UI.</i></h3>
+                    <img alt="logo" height="30" src="https://peviitor.ro/static/media/peviitor_logo.df4cd2d4b04f25a93757bb59b397e656.svg" width="100"/>
+                </div>
+            </a>
+            <div class="nav-links">
+                <a href="https://firme.peviitor.ro/" target="_blank">Firme PeViitor</a>
+                <a href="../../doc.html">Documentation</a>
+            </div>
+        </div>
+        <!-- Start Section Alert -->
+        <div class="alertPopUp active">
+            <div class="alertPopUp-content">
+                <img alt="Check" class="alert-img" id="image-alert" src="" />
+                <div class="message">
+                    <span class="alert-text-1"> </span>
+                    <span class="alert-text-2"> </span>
+                </div>
+            </div>
+            <p class="close">X</p>
+            <div class="progress active"></div>
+        </div>
+        <!-- End Section Alert -->
+        <section class="company">
+            <div class="company-container">
+                <div class="flip-card-container">
+                    <div class="flip-card">
+                        <div class="box">
+                            <img alt="sp"  src="None" />
```

Comment:
you forgot image url
```suggestion
                            <img alt="sp"  src="image_url" />
```